Queueomatic/Server/Endpoints/User/Delete/DeleteUserEndpoint.cs

```diff
@@ -1,9 +1,16 @@
 ﻿using FastEndpoints;
+using Queueomatic.DataAccess.UnitOfWork;
 
 namespace Queueomatic.Server.Endpoints.User.Delete;
 
 public class DeleteUserEndpoint: Endpoint<DeleteUserRequest, DeleteUserResponse>
```

Comment:
As Fabian commented on return value, you can just have a request. Take a look at FastEndpoints doc and APIspec in wiki.